gravitee-apim-console-webui/src/management/application/details/general/application-general.component.ts

```diff
@@ -159,4 +165,35 @@ export class ApplicationGeneralComponent implements OnInit {
       )
       .subscribe(() => this.ngOnInit());
   }
+
+  deleteApplication() {
+    this.matDialog
+      .open<GioConfirmAndValidateDialogComponent, GioConfirmAndValidateDialogData>(GioConfirmAndValidateDialogComponent, {
+        width: '500px',
```

Comment:
You can use the particles value: `GIO_DIALOG_WIDTH.MEDIUM`

Reply:
Done